src/components/List/List.js

```diff
@@ -23,7 +23,6 @@ export default function List({ habits, onItemClick }) {
 }
 
 const CurrentDate = styled.div`
-  margin-top: 20px;
   text-align: center;
 `
 
```

Comment:
Your `HabitList` is called `List` maybe consider using an HTML list element instead of a section?